README.md

```diff
@@ -192,6 +192,29 @@ Example:
 GATSBY_LAUNCH_SRC=https://your.adobe.launch.url.here
 GATSBY_LAUNCH_SRC_INCLUDE_IN_DEVELOPMENT=true
 ```
+### Publishing to GitHub Pages (Dev)
+
+On every commit to the `master` branch, the site will be built to GitHub Pages automatically, for you to preview as a development version. This is the default for new repos in GitHub: on Oct 1st, this will [default to main](https://github.blog/changelog/2020-08-26-set-the-default-branch-for-newly-created-repositories/)
+
+`GitHub Contributors component`: this will use the GitHub token automatically provided by the GitHub Action to retrieve data
+
+`Feedback component`: no environmental variable should be set since GitHub Pages should only be for development purposes
+
+### Publishing to Azure Storage Static Websites (Production)
+
+A site is published via a Pull Request:
+1. The Pull Request must be `approved`
+2. The Pull Request should be tagged with the `deploy` label (Production deploy) AND/OR
+3. The Pull Request should be tagged with the `deploy:dev` label (Dev deploy)
+
+**Pre-requisites:**
+1. Create a GitHub label called `deploy` if it does not exist
+2. Create a GitHub label called `deploy:dev` if it does not exist
+3. Add a Azure Blob Storage connection string GitHub Secret for `production` called `AZURE_PROD_CONNECTION_STRING`
+4. Add a Azure Blob Storage connection string GitHub Secret for `development` called `AZURE_DEV_CONNECTION_STRING`
+4. Add a Adobe Launch URL GitHub Secret called `GATSBY_LAUNCH_URL` (see section above for the Feedback component)
```

Comment:
@shazron want to set these 3 secrets at the org level so folks don't have to add them manually?